module/VuFind/src/VuFind/Bootstrapper.php

```diff
@@ -354,5 +354,8 @@ protected function initContentSecurityPolicy(): void
         if ($cspHeader = $cspHeaderGenerator->getHeader()) {
             $headers->addHeader($cspHeader);
         }
+        if ($reportToHeader = $cspHeaderGenerator->getReportToHeader()) {
+            $headers->addHeader($reportToHeader);
+        }
```

Comment:
Arguably there should be a separate generator class for this (separate) HTTP header, and a separate .ini file, but that seemed a lot of complexity to add given this header is only used (for now at least) in a CSP context, and the configurations have to match up.

Reply:
Yes, I support keeping this in one place.

Reply:
I suppose a possible refactoring here could be to add a `getHeaders` method to the helper that returns an array with both of the relevant headers, and then iterate over the result of that in the bootstrapper. We could then deprecate the existing getHeader method in favor of something more descriptive like getCspHeader. This would allow us to add further CSP-related headers in the future without having to make additional changes to the bootstrapper.

Reply:
I think the problem here is context; right now it's fair to call both of these headers (and the unused Reporting-Endpoints) as "csp headers", but I think the intent from W3C etc. is that ultimately the reporting headers would be used in concert with several other non-CSP headers.  And some of those things may be easier for implementers to adopt than CSP itself.  So I would hesitate to tie reporting closer together in the CSP bucket than it already is.

Reply:
The way I'm thinking about this is that the CSP header generator is providing all the headers we need to make CSPs work correctly. You could argue that using a generic getHeaders method actually ties the reporting header less directly with the CSP functionality since it's abstracted away instead of directly requested, and it could be moved elsewhere with less disruption to external code, if we find that we eventually do want a separate class to manage reporting-related headers.

Reply:
Ok, that make sense.  I did the refactor.

Reply:
> You could argue that using a generic getHeaders method actually ties the reporting header less directly with the CSP functionality since it's abstracted away instead of directly requested, and it could be moved elsewhere with less disruption to external code, if we find that we eventually do want a separate class to manage reporting-related headers.

FYI I decided last night that I'm going to follow this up with a PR that adds the NEL header (Network-Error-Logging).  So this will be non-hypothetical pretty quick.  Still not sure whether actually moving Report-To (and NEL) to separate classes is a good idea, but we can consider it in that PR.